covin_slot_tracker.py

```diff
@@ -61,52 +67,56 @@ def checkAvailability(payload, age):
     unavailable_centers = set()
     available_centers_str = False
     total_available_centers = 0
-    
+
     if('centers' in payload.keys()):
-       length = len(payload['centers'])
-       if(length>1):
+        length = len(payload['centers'])
+        if(length > 1):
             for i in range(length):
                 sessions_len = len(payload['centers'][i]['sessions'])
                 for j in range(sessions_len):
-                    if((payload['centers'][i]['sessions'][j]['available_capacity']>0) and
-                       (payload['centers'][i]['sessions'][j]['min_age_limit']<=age)):
+                    if((payload['centers'][i]['sessions'][j]['available_capacity'] > 0) and
+                       (payload['centers'][i]['sessions'][j]['min_age_limit'] <= age)):
                         available_centers.add(payload['centers'][i]['name'])
-            available_centers_str =  ", ".join(available_centers)
+            available_centers_str = ", ".join(available_centers)
             total_available_centers = len(available_centers)
-    
-    return available_centers_str,total_available_centers
+
+    return available_centers_str, total_available_centers
 
 
-if __name__=="__main__":
+if __name__ == "__main__":
     with open("settings.json") as f:
         settings = json.load(f)
 
     # Load from JSON file
-    key_list = ["districtId","authToken","accountSID","twilioPhone","selfPhone","userAge"]
+    key_list = ["districtId", "authToken", "accountSID",
+                "twilioPhone", "selfPhone", "userAge"]
     if all(key in settings for key in key_list):
         DISTRICT_ID = settings["districtId"]
         SECRET_TOKEN = settings["authToken"]
         ACCOUNT_SID = settings["accountSID"]
         TWILIO_PHONE_NUMBER = settings["twilioPhone"]
         CELL_PHONE_NUMBER = settings["selfPhone"]
         USER_AGE = int(settings["userAge"])
-        
+
         client = Client(ACCOUNT_SID, SECRET_TOKEN)
         while(True):
-            date = getDate()
-            data1 = pingCOWIN(date,DISTRICT_ID)
-            available, total_centers = checkAvailability(data1,USER_AGE)
-            if available:
-                msg_body = "Slots Available at {total} places.\n{available}".format(total = total_centers,available = available)
-                print(msg_body)
-                client.messages.create(from_=TWILIO_PHONE_NUMBER,
-                           to=CELL_PHONE_NUMBER,
-                           body= msg_body)
-            else:
-                print("No Available Centers")
-            
+            smsSent = False
```

Comment:
Also could you stick to one type of naming convention

Reply:
> What is the use of smsSent?

It's checking for 90 days in a go so I don't want 90 SMSs for that. It will send one sms for "90 days (90 iterations)" then after it for loop and goes for next iteration of while it will again send sms

Reply:
> Also could you stick to one type of naming convention

I come from Unity3D, this happens. Will do.